operator/deploy/operator/operator.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: apicurio-registry-operator
+  namespace: PLACEHOLDER_NAMESPACE
+spec:
+  replicas: 1
+  selector:
+    matchLabels: { }
+  template:
+    metadata:
+      labels: { }
+    spec:
+      containers:
+        - name: apicurio-registry-operator
+          image: PLACEHOLDER_IMAGE
+          imagePullPolicy: Always
+          env:
```

Comment:
double check those variables

Reply:
Done, we only need `QUARKUS_OPERATOR_SDK_CONTROLLERS_APICURIOREGISTRY3RECONCILER_NAMESPACES` to support the OLM. This means the operator not deployed by OLM watches all namespaces by default. We can add an own-namespace support or a more complex configuration later.